packages/db/scripts/migration-smoke.mjs

```
@@ -0,0 +1,41 @@
+  order by table_name;
+`);
+
+const actual = new Set(result.rows.map((row) => row.table_name));
+for (const table of expected) {
+  if (!actual.has(table)) {
+    throw new Error(`missing table after migration: ${table}`);
+  }
+}
+
+await pool.end();
+console.log('migration smoke check passed');
```

ESLint

[error] 'console' is not defined.

(no-undef)